# point_sim_controller.py and point_planner.py config file

sim:
  update_freq: 25
  random_pose_min_max: [-100, 100]
  random_yaw: false
  
active_auvs:
  - ava

ava:
  start_pos:           # either x,y,z,psi or 'random'
    x: 0
    y: 0
    z: 0
    psi: 0
  start_twist:
    x: 0.0
    y: 0.0
    z: 0
    psi: 0
sensor_range: 2
sensor_noise_std: 2

landmarks:
  A: [-4, -2]
  B: [-4, 1]
  C: [-3, 3]
  D: [-2, -1]
  E: [-1, -3]
  F: [0, 3]
  G: [1, 1]
  H: [2, 3]
  I: [2, -2]
  J: [4, 2]
  K: [4, -1]